Buildings/Controls/OBC/ASHRAE/PrimarySystem/ChillerPlant/Tower/FanSpeed/EnabledWSE/Subsequences/Validation/WSEOperation.mo

```diff
@@ -0,0 +1,94 @@
+within Buildings.Controls.OBC.ASHRAE.PrimarySystem.ChillerPlant.Tower.FanSpeed.EnabledWSE.Subsequences.Validation;
+model WSEOperation
+  "Validation sequence of controlling tower fan speed when waterside economizer is running alone"
```

Comment:
Please check the consistency of the output of the test after revising the model tested as suggested.